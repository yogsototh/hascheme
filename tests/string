"This is a simple string"
String "This is a simple string"
"This is a string with a return ->\n<- Here"
String "This is a string with a return ->\n<- Here"
"More difficult string \n, \r, \t, \\, \x, \A"
String "More difficult string \n, \r, \t, \\, x, A"
