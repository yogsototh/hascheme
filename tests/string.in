"This is a simple string"
