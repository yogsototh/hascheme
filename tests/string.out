String "This is a simple string"
